addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -447,6 +449,13 @@ def migration_invoice_moves(env):
             WHERE debit IS NULL
             """,
     )
+    openupgrade.logged_query(
```

Comment:
Aren't we able to inject this value on the UPDATE/INSERT queries?

Reply:
well, there are several "insert into" queries. do you want to put that in all of them instead?

Reply:
Yes. Remember my last IMP PR to account, where this kind of queries were spending 13 minutes for 2.2M records, so it's worth to include it in the others.